utils/build/webpack.common.js

```diff
@@ -81,9 +84,29 @@ const withStylingModuleLoader = returnModuleRuleWithConfig(
   ]
 );
 
+const withTSModuleLoader = (tsConfigFile) =>
+  returnModuleRuleWithConfig(
+    {
+      test: /\.(tsx|ts)?$/,
```

Comment:
Are we going to allow js files at all if not then this is fine.

Reply:
This is configuration for the `ts-loader`, which included in the dev/prod configs for webpack as needed. atm both JS and TS loaders are included - I will see if we can go solely with one over another